src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -802,7 +822,11 @@ namespace AppInstaller::CLI::Workflow
                     );
 
                     auto pinnedState = ConvertToPinTypeEnum(installedVersion->GetMetadata()[PackageVersionMetadata::PinnedState]);
-                    if (m_onlyShowUpgrades && pinnedState == PinType::PinnedByManifest)
+                    if (updateIsPinned)
+                    {
+                        linesForPins.push_back(std::move(line));

```

Comment:
Will this show PinnedByManifest in both the blocking/gating and explicit upgrade table?

Reply:
No, each package is in only one table. Despite my original intentions, the handling for PinnedByManifest and user pins ended up being totally independent, so changes in pinning wouldn't affect that.

Reply:
> No, each package is in only one table. Despite my original intentions, the handling for PinnedByManifest and user pins ended up being totally independent, so changes in pinning wouldn't affect that.

Perfect, thanks for clarifying!